src/RA_Core.cpp

```diff
@@ -734,52 +748,52 @@ API int CCONV _RA_HandleHTTPResults()
 //	Following this function, an app should call AppendMenu to associate this submenu.
 API HMENU CCONV _RA_CreatePopupMenu()
 {
-    HMENU hRA = CreatePopupMenu();
-    HMENU hRA_LB = CreatePopupMenu();
+    // Callers need to claim ownership of hRA, hRA_LB is automatically deleted
+    ra::MenuH hRA{ ::CreatePopupMenu() };
+    ra::MenuH hRA_LB{ ::CreatePopupMenu() };
```

Comment:
The Initlizer syntax is required. `std::unique_ptr'`s constructors are explicit. Notice how we didn't need to specify the delete function.
`explicit unique_ptr( pointer p ) noexcept;`